docs/references/http-websocket-apis/public-api-methods/ledger-methods/ledger_entry.md

```diff
@@ -921,6 +923,200 @@ An example of a successful response:
 
 {% /tabs %}
 
+### Get MPT Issuance Object 
+
+Return an `MPTokenIssuance` object.
+
+| Field                   | Type                       | Description           |
+|:------------------------|:---------------------------|:----------------------|
+| `mpt_issuance`              | String | The 192-bit `MPTokenIssuanceID` that's associated with the MPTokenIssuance. |
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+    "id": "example_get_mpt_issuance",
+    "command": "ledger_entry",
+    "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E",
+    "ledger_index": "validated"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+  "method": "ledger_entry",
+  "params": [{
+    "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E",
+    "ledger_index": "validated"
+  }]
+}
+```
+*Commandline*
+
+```sh
+rippled json ledger_entry '{ "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E", "ledger_index": "validated" }'
+```
+### Get MPToken Object 
+
+Return an `MPToken` object.
+
+| Field                   | Type                       | Description           |
+|:------------------------|:---------------------------|:----------------------|
+| `mptoken`               | ️Object or String          | If a string, interpret as ledger entry ID of the MPToken to retrieve. If an object, requires the sub-fields account and mpt_issuance_id to unique identify the MPToken. |
+| mptoken.mpt_issuance_id |	️String                      |	(Required if the `MPToken` is specified as an object) The 192-bit MPTokenIssuanceID that's associated with the MPTokenIssuance. |
+| mptoken.account	️         | String	                    | (Required if the `MPToken` is specified as an object) The account that owns the MPToken. |
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+    "id": "example_get_mpt_issuance",
+    "command": "ledger_entry",
+    "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E",
+    "ledger_index": "validated"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+  "method": "ledger_entry",
+  "params": [{
+    "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E",
+    "ledger_index": "validated"
+  }]
+}
+```
+*Commandline*
+
+```sh
+rippled json ledger_entry '{ "mpt_issuance": "000004C463C52827307480341125DA0577DEFC38405B0E3E", "ledger_index": "validated" }'
```

Comment:
these examples are for querying MPTokenIssuance object, not MPToken. Below is an example of querying an MPToken object
```

    "method": "ledger_entry",
    "params": [
        {
            "mptoken":{
                "mpt_issuance_id": "000002DFA4D893CFBC4DC6AE877EB585F90A3B47528B958D",
                "account":"r33kves44ksufkHSGg3M6GPPAsoVHEN8C1"
            } 
        }
    ]
```